tests/test_grains.py

```diff
@@ -423,6 +492,7 @@ def test_find_grains(
     smallest_grain_size_nm2: int,
     remove_edge_intersecting_grains: bool,
     expected_grain_mask: npt.NDArray[np.int32],
+    expected_labelled_regions: npt.NDArray[np.int32],
```

Comment:
The types here are `np.int32` but in the above parametrisation the arrays are explicitly set to be `np.int64`.

Reply:
Solved 😃